ui-particles-angular/projects/ui-particles-angular/src/lib/gio-emty-state/gio-empty-state.component.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'gio-empty-state',
```

Comment:
: Note sure :🙈 But what about name it like `gio-card-empty-state` if we only use it inside card ?

My idea it's to prefix each "card component" with `gio-card-*`